.golangci.yml

```diff
@@ -1,8 +1,55 @@
+run:
+  timeout: 5m
+  modules-download-mode: vendor
```

Comment:
Mb, this option not required here.

Reply:
Yes, it's not required here and can be safely deleted.